tests/common/widgets/autocomplete/options.tsx

```diff
@@ -140,7 +140,7 @@ export function createOptionsTests(
       });
 
       const callTimes: Record<string, Record<SupportedFlavor, number>> = {
-        initial: { javascript: 2, react: 4, vue: 0 },
+        initial: { javascript: 2, react: 2, vue: 0 },
         refined: { javascript: 1, react: 2, vue: 0 },
```

Comment:
Ideally for react this should also only be 1 instead of 2 like in js but not sure why this is the case still. I don't think it's related to changes in this PR so should be investigated later.